doc/antora/modules/concepts/pages/modules/ldap_authentication.adoc

```diff
@@ -52,7 +52,8 @@ more detail.
 
 == Password Storage Methods
 
-If the `userPassword` field is returned from LDAP to FreeRADIUS, that
+If the `userPassword` field is retu:/raddb
```

Comment:
?  typo here?

you can edit the file, then do

```
git commit --amend doc/antora/modules/ROOT/pages/index.adoc
git push -f ...
````
and the PR will be updated